spec/models/firm_factory_spec.rb

```diff
@@ -0,0 +1,189 @@
+RSpec.describe 'Firm factory' do
+  subject do
+    begin
+      FactoryGirl.create(factory)
+    rescue ActiveRecord::RecordInvalid
```

Comment:
... and the explicitness is then provided by doing a `expect(subject).to be_persisted`.